sql/moz-fx-data-shared-prod/udf/monetize_search_v1/query.sql

```diff
@@ -0,0 +1,146 @@
+/*
+
+Return a boolean value indicating whether the search is monetized
+
+*/
+CREATE OR REPLACE FUNCTION udf.monetized_search(
```

Comment:
sounds good, let's gather all the review and comments here, and once it's finalized, I will create the udf there.

Reply:
@xluo-ds, in the main directory, you could also do a `mkdir sql/search` then a `git mv udf/monetize_search_v1.query.sql sql/search/monetized_search.sql` and it will retain the history, etc.

Reply:
One further suggestion on naming: Part of the reason for creating a separate `mozfun` project was so that we could use the dataset part of the name to convey meaning about what the function does. In that spirit, I would `mozfun.search.is_monetized` which removes a redundant use of `search` and conveys that this function returns a boolean value.